main.tex

```diff
@@ -331,21 +335,21 @@ \section*{Introduction}
 % BioModels\footnote{\url{https://www.ebi.ac.uk/biomodels/}}, JWS Online\footnote{\url{https://jjj.biochem.sun.ac.za/}}, the CellML Model Repository\footnote{\url{https://models.cellml.org/}}
-For example, querying the BioModels database for biochemical and cellular simulation models that contain proteins such as \wnt{}, Janus kinase (Jak), or mitogen-activated protein kinase (MAPK), which are associated with corresponding signaling pathways, returns for \wnt{} a set of 22 simulation models, for Jak a set of 12 simulation models and for MAPK a set of 139 simulation models (as of January 2021).
+For example, querying the BioModels database for biochemical and cellular simulation models that contain proteins such as \wnt{}, Janus kinase (Jak), or mitogen-activated protein kinase (MAPK), which are associated with corresponding signaling pathways, returns 22 simulation models for \wnt{}, for 12 simulation models for Jak and 139 simulation models for MAPK (as of January 2021).
 This already shows that MAPK is an intensively studied signaling pathway.
 However, there is no way to easily compare these simulation models or to examine their relationships to each other.
-Tools such as BiVeS~\cite{Scharm2016a} are helpful to compare different versions of one particular simulation model, but comparing different models---even of the same system---will be a difficult task because the syntax of these models (e.g., the names of the species), as well as reactions, might be completely different.
+Tools such as BiVeS~\cite{Scharm2016a} are helpful to compare different versions of one particular simulation model, but comparing different models---even of the same system---is a difficult task because the syntax of these models (e.g., the names of the species), as well as reactions, might be completely different.
 Instead of analyzing the similarities and differences in the specifications of simulation models to infer possible relationships between these simulation models, we will focus on context information, such as how a simulation model has been generated.
 
 % How do models grow larger over time and what happens then?
 Particularly, larger models are usually not built from scratch~\cite{Cvijovic2014}.
-In general, simulation models are the outcome of extensive as well as interactive model and data generation activities that include, in addition to executing various simulation experiments and successive model refinements, the adaptation of already existing models, for example, by composition or extension~\cite{Peng2017, Peng2016, Cvijovic2014}.
+In general, simulation models are the outcome of extensive, as well as interactive, model and data generation activities. These include, in addition to executing various simulation experiments and successive model refinements, the adaptation of already existing models, for example, by composition or extension~\cite{Peng2017, Peng2016, Cvijovic2014}.
```

Comment:
Split up this larger sentence into two. I think this makes it easier easier to follow?